page3.html

```diff
@@ -46,23 +46,44 @@
 		?>
 	</div>
 	
+	<?php
+		require("13csi_Database_Assessment_mysqli.php")			
+	?>
+	
 	<div class="header">
 		<h1>Z U L U</h1>
 	</div>
 	<br>
+
 	<div class="content">
-		<h3>Top Songs:</h3>
-		<hr>
-		<div class="image"><img src="images/image_placeholder3.jpg" width="99.7%"></div>
-		<hr>
-		<div class="image"><img src="images/image_placeholder3.jpg" width="99.7%"></div>
-		<hr>
-		<div class="image"><img src="images/image_placeholder3.jpg" width="99.7%"></div>
-		<hr>
-	</div>
-	<!-- Copyright Statement -->
-	<div class="footer">
-		<p>&copy; Copyright Jake Boyd, all rights reserved, Tawa College 2022</p>
-	</div>	
+		<h3>Top Songs:</h3><br>

```

Comment:
instead of placeholder images the images are now sourced from the database and are also clickable links which take the user to a separate page (song.php) with more information about the song